repo/packages/P/prometheus/0/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "prometheus",
+  "version": "0.15.1-0.1",
+  "scm": "https://github.com/prometheus/prometheus",
+  "maintainer": "support@mesosphere.io",
```

Comment:
this has to be a support email that isn't mesosphere :)  even if it is a dev list in the open source space.